pkgs/applications/audio/feishin/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchurl
+, appimageTools
+, pkgs
+}:
+
+let
+  pname = "feishin";
+  version = "0.0.1-alpha5";
+  src = fetchurl {
+    url = "https://github.com/jeffvli/feishin/releases/download/v${version}/Feishin-${version}-linux-x86_64.AppImage";
+    sha256 = "sha256-YLs3aL8VAM15oHNm+dOHvSrqByBJ4fiUqDOt0AaqscQ=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 rec {
```

Comment:
Is this `rec` needed? Given that all possibly recursive references are `let`-enclosed...